lib/tasks/grda_warehouse.rake

```diff
@@ -330,6 +330,11 @@ namespace :grda_warehouse do
       Hmis::AutoExitJob.perform_now
     end
 
+    if DateTime.current.hour == 5 && HmisEnforcement.hmis_enabled? && GrdaWarehouse::DataSource.hmis.exists? && Hmis::Ce.configuration.enabled?
```

Comment:
Question: How to test this change? Just on QA, or is there anything I can verify locally to make sure the change is good?

Reply:
QA is probably the easiest, this config should be fine.  The only thing I might change, is that if you expect `perform_now` to take a long time to run below you should queue it with `perform_later` so it doesn't prevent other things from running.